rootex/framework/scene.cpp

```diff
@@ -302,6 +302,20 @@ bool Scene::removeChild(Scene* toRemove)
 	return false;
 }
 
+bool Scene::isReservedName(String sceneName)
+{
+	Vector<String> reserved_names_list { "pause", "pauseUI" };
+	for (auto& reservedName : reserved_names_list)
+	{
+		if (reservedName == sceneName)
+		{
+			WARN("Cannot use reserved Scene names");
```

Comment:
Take care of Casing 
`Cannot use reserved scene names`

Reply:
Fixed